Task_TheGridProblem/index.css

```diff
@@ -1 +1,72 @@
-/* Start coding below for GRID PROBLEM TASK */
+/* Start coding below for GRID PROBLEM TASK */
+ * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      body {
+        display: grid;
+        place-items: center;
+      }
+
+      .container {
+        color: #fff;
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 1fr 0.5fr 1fr;
+        gap: 10px;
+        height: 700px;
+        width: 700px;
+        padding: 20px;
+      }
+
+      .item-1 {
+        grid-column: 1 / 2;
+        grid-row: 1 / 2;
+      }
+
+      .item-2 {
+        grid-column: 2 / 3;
+        grid-row: 1 / 2;
+      }
+
+      .item-3 {
+        grid-column: 3 / 3;
+        grid-row: 1 / 2;
+      }
+
+      .item-4 {
+        grid-column: 1/ 3;

```

Comment:
The grid-row property on line 40 should be '4 / 3' instead of '4 / 2' to specify the correct row range.